### ISSUE TITLE
Increasing color contrast of input fields

### ISSUE THREAD

**Issue by runningonsideprojects** [processing/p5.js-web-editor#3944]
### Increasing Access

adhering more components to WCAG guidelines ensures that more people are able to use the editor

## Issue Description

This specific subissue aims to increase the contrast of several input field elements so they adhere to WCAG guidelines. WCAG guidelines [state](https://www.allaccessible.org/blog/color-contrast-accessibility-wcag-guide-2025) that non-text color contrast should be at least 3:1, and small text contrast should be 4.5:1.

The elements we’ve identified that are failing to adhere to these provisions are: 
- search sketches text field
- checkboxes in the settings modal
- name field when creating a file in the editor.

In light mode: 
- The fields are not distinguishable from the background because there is neither:
  - 3:1 contrast between the input background color (`$lightest`) and the page background
  - NOR 3:1 contrast between the field outline and the page background 
- The placeholder text color is also too low contrast with the field background

In dark mode: 
- The fields are not distinguishable from the background because there is neither:
  - 3:1 contrast between the field background color (`$dark`) and the page background
  - NOR 3:1 contrast between the field outline and the page background 
- And the color contrast between the placeholder text (`#757575`) and input background color ($dark) of the input element is 2.74:1

## Change Requested
- Light theme:
  - [ ] change `input-border-color` to `$middle-dark`
  - [ ] Configure `input::placeholder` (variable that sets inactive input text) to `$middle-gray`

- Dark theme: 
  - [ ] Change `input-border-color` to either `$light `or `$lighter` 
  - [ ] Change Configure `input::placeholder` (variable that sets inactive input text) to `$middle-gray`

## Visuals

Location of the Search sketches component in the editor: [https://editor.p5js.org/{username}/sketches](https://editor.p5js.org/alisonkim12/sketches)

<img width="512" height="123" alt="Image" src="https://github.com/user-attachments/assets/cec9ba1c-8f3f-40bd-859c-6bd4d7ed3cb8" />

<img width="512" height="111" alt="Image" src="https://github.com/user-attachments/assets/d301247a-d078-4edb-bec6-d2b2f2a77685" />

Location of the Create File component: 1) click the < arrow next to [sketch.js](http://sketch.js/), then a drawer should open up. 2) Click the + next to Sketch Files.

<img width="512" height="380" alt="Image" src="https://github.com/user-attachments/assets/fbc1d282-4ae7-4cb0-a889-c6adbfd638ef" />

<img width="512" height="156" alt="Image" src="https://github.com/user-attachments/assets/1029d4dd-e52f-4b70-bfcc-c29cbd0f1f29" />

<img width="512" height="160" alt="Image" src="https://github.com/user-attachments/assets/c60c5b2c-6032-498f-a4d0-8f53e4510456" />

Location of the checkboxes in the Settings Modal: 

https://editor.p5js.org/ (sketch files > + > create file and settings > accessibility)

<img width="512" height="180" alt="Image" src="https://github.com/user-attachments/assets/05d05de5-7f51-4ee0-9d37-a95078b9de18" />

## Files to change
- [ ] input-border-color and input-background-color in variables.scss 

## PR Criteria
- [ ] Provide screenshots in ur PR of the visual changes implemented
- [ ] Provide QA steps for how reviewers can ensure your changes are made properly
- [ ] Ensure that the changes to input-border-color and input-background-color do not cause contrast issues for other elements.

Additional Resources:
 
- [Color Combinations for Accessibility](https://github.com/processing/p5.js-web-editor/wiki/Color-Combinations-for-Accessibility) – tool developed by @izzy-snyder that generates custom color palettes for components while adhering to WCAG guidelines and [p5.js](http://p5.js/) styling standardizations. 
- [Zeplin Style Link](https://scene.zeplin.io/project/55f746c54a02e1e50e0632c3) – the p5.js editor’s styling standardizations
- [Preparing a Pull Request](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/preparing_a_pull_request.md) – how-to guide on creating a Pull Request 
- [Style Guide Documentation](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/styling.md) (WIP), best-practices guide on implementing styling changes to the codebase.

**Comment by dpoppe7** [comment]
Hello! I would like to work on this issue, could it be assigned to me?
